app/timetables/admin.py

```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
+from app.timetables.models import MealOption
```

Comment:
Add an extra line above here, we would want to visually separate imports from django and imports from the app